docs/content/integrations/airflow/migrating-to-dagster.mdx

```diff
@@ -13,99 +13,208 @@ description: guide for doing a lift and shift migration of airflow to dagster
   !
 </Note>
 
-Dagster can convert your Airflow DAGs into Dagster jobs, enabling a lift-and-shift migration from Airflow to Dagster without any rewriting.
+Dagster can convert your Airflow DAGs into Dagster jobs, enabling a lift-and-shift migration from Airflow without any rewriting.
 
-This guide will walk you through the steps of doing this migration.
+This guide will walk you through the steps of performing this migration.
 
 ---
 
 ## Prerequisites
 
-To get started, you will need to install the `dagster` and `dagster-airflow` Python packages:
+Migrations are complicated as there are lots of moving pieces. It's important for you to go in knowing exactly how much scope and complexity you'll be tackling.
+
+To complete the migration, you'll need:
+
+- **To understand the complexity of undertaking a migration**. Refer to the [previous section](#before-you-begin) for more detail.
+
+- **To know the following about your Airflow setup**:
+
+  - What operator types are used in the DAGs you're migrating
+  - What Airflow connections your DAGs depend on
+  - What Airflow variables you've set
+  - What Airflow secrets backend you use
+  - Where the permissions that your DAGs depend on are defined
+
+  If you're unsure where to find these things, you're likely not the right person to make the migration. We recommend looping a teammate in before continuing.
```

Comment:
I think we can drop "If you're unsure where to find these things, you're likely not the right person to make the migration. We recommend looping a teammate in before continuing."